Monika After Story/game/script-holidays.rpy

```diff
@@ -6899,22 +6899,35 @@ image mas_bday_balloons = MASFilterSwitch(
 
 ############### [HOL060]: METHODS
 init -1 python:
-    def mas_isMonikaBirthday(_date=None):
+    def mas_isMonikaBirthday(_datetime=None, _extend=0):
         """
         checks if the given date is monikas birthday
-        Comparison is done solely with month and day
 
         IN:
-            _date - date to check. If not passed in, we use today.
+            _datetime - datetime to check. If not passed in, we use now.
+            _extend - int of how many hours we want to extend past 922
+                defaults to 0
         """
-        if _date is None:
-            _date = datetime.date.today()
+        if _datetime is None:
+            _datetime = datetime.datetime.now()
```

Comment:
Shouldn't we use `datetime.datetime.utcnow`?

Reply:
using localtime for bday checking makes sense since the time that the player sees is more appropriate than their absolute time.

However, this should be a separate function (like `mas_isMonikaBirthday_dt`) since changing this function's input will break submods that use it with an input date.